spark-indexer/src/main/java/org/uniprot/store/spark/indexer/literature/LiteratureDocumentsToHDFSWriter.java

```diff
@@ -0,0 +1,96 @@
+package org.uniprot.store.spark.indexer.literature;
+
+import static org.uniprot.store.spark.indexer.common.util.SparkUtils.getCollectionOutputReleaseDirPath;
+import static org.uniprot.store.spark.indexer.literature.mapper.LiteratureEntryStatisticsJoin.*;
+import static org.uniprot.store.spark.indexer.publication.MappedReferenceRDDReader.*;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.uniprot.core.citation.Literature;
+import org.uniprot.core.literature.LiteratureEntry;
+import org.uniprot.store.search.SolrCollection;
+import org.uniprot.store.search.document.literature.LiteratureDocument;
+import org.uniprot.store.spark.indexer.common.JobParameter;
+import org.uniprot.store.spark.indexer.common.util.SolrUtils;
+import org.uniprot.store.spark.indexer.common.writer.DocumentsToHDFSWriter;
+import org.uniprot.store.spark.indexer.literature.mapper.*;
+import org.uniprot.store.spark.indexer.publication.MappedReferenceRDDReader;
+import org.uniprot.store.spark.indexer.uniprot.UniProtKBRDDTupleReader;
+
+/**
+ * @author lgonzales
+ * @since 24/03/2021
+ */
+@Slf4j
+public class LiteratureDocumentsToHDFSWriter implements DocumentsToHDFSWriter {
+
+    private final JobParameter parameter;
+
+    public LiteratureDocumentsToHDFSWriter(JobParameter parameter) {
+        this.parameter = parameter;
+    }
+
+    @Override
+    public void writeIndexDocumentsToHDFS() {
+        LiteratureRDDTupleReader literatureReader = new LiteratureRDDTupleReader(parameter);
+
+        // load literature with abstract JavaPairRDD<citationId, Literature>
+        JavaPairRDD<String, Literature> literature = literatureReader.load();
+
```

Comment:
To be consistent with other variable name, we should call literature as literatureRDD